app/commands/markdown/render_html.rb

```diff
@@ -18,13 +18,26 @@ def initialize(options:, nofollow_links: false)
     attr_reader :nofollow_links
 
     def link(node)
-      out('<a href="', node.url.nil? ? '' : escape_href(node.url), '" target="_blank"')
+      out('<a href="', node.url.nil? ? '' : escape_href(node.url), '"')
       out(' title="', escape_html(node.title), '"') if node.title.present?
+      if external_url(node.url)
+        out(' target="_blank"')
+        out(' rel="noopener"') unless nofollow_links
```

Comment:
You can't have two `rel` attributes. Do we want `rel="noopener"` to override `rel="nofollow"`? For now I've opted for the latter to override the first.